test_haystack/whoosh_tests/test_whoosh_backend.py

```diff
@@ -17,15 +18,19 @@
 from haystack.query import SQ, SearchQuerySet
 from haystack.utils.loading import UnifiedIndex
 
-from ..core.models import AFourthMockModel, AnotherMockModel, MockModel
-from ..mocks import MockSearchResult
-from .testcases import WhooshTestCase
+from test_haystack.core.models import AFourthMockModel, AnotherMockModel, MockModel
```

Comment:
This change shouldn't have been necessary to get it to run — did you change it for aesthetic reasons or was it actually breaking local tests?

Reply:
My bad, I think that change slipped in because I was trying to help PyCharm resolve some warnings or errors. I'll revert this change!